registry/accessing-the-registry.adoc

```diff
@@ -49,8 +49,10 @@ include::modules/registry-viewing-logs.adoc[leveloffset=+1]
 
 include::modules/registry-accessing-metrics.adoc[leveloffset=+1]
 
-.Additional resources
+include::modules/
```

Comment:
This `include` line does not contain a complete relative path to a module file. Is your intention to include another module here?

* If so, please add the relative path to the target module file, followed by a `leveloffset` heading indentation specification. Here is an example from an earlier line in the same assembly:

`include::modules/registry-accessing-metrics.adoc[leveloffset=+1]`

* If not, please remove the line because it is rendering in then preview as "include::modules/".

Reply:
@pneedle-rh mea culpa-I should've removed this when I decided against including.